### PR TITLE
Ped

### LINKED ISSUE #153: [PE-D] Documentation in Section 3.1.4 in UG
Should it be project /list instead?

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: EdmundEXE/ped#2

### LINKED ISSUE #157: [PE-D] Command in Quick Start Section does not work
The command `project /create -title Scrumptious -desc A java project -end 30 -sd 10` did not successfully create a project but shows an error message: Missing parameters.

Actual output: 

Expected output from user guide: 

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: KennethEer/ped#1

### LINKED ISSUE #158: [PE-D] Command to create project behaves differently from expected output
The expected output of the command to `create project` is `Project successfully created.`
 
However, the actual output shows Project successfully created together with the details of the project. 

The details of the project should only be shown when the user types project /view.

Actual output: 

Expected output from user guide: 

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: KennethEer/ped#2

### LINKED ISSUE #165: [PE-D] Formatting Issue
On the Quick Start section, the -end tags doesn't seem to work. Supposed to be -dur?

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: jusufnathanael/ped#1

### LINKED ISSUE #166: [PE-D] Formatting Issue
Supposed to be project /list I guess.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: jusufnathanael/ped#2

### LINKED ISSUE #168: [PE-D] UI message
Maybe you don't want to print "The details are as follows..." if the input is invalid.

I think you can also implement like task /view 1-3 that takes a range of numbers as the input.

-------------
Labels: `severity.VeryLow` `type.FunctionalityBug`
original: jusufnathanael/ped#4

### LINKED ISSUE #171: [PE-D] Formatting Issue
Do you perhaps want to sort the list from the lower index first?

-------------
Labels: `severity.VeryLow` `type.FunctionalityBug`
original: jusufnathanael/ped#7

### LINKED ISSUE #172: [PE-D] UG description bug
The command at introduction is not valid the it differs from the command at the later part of the UG.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: JuZihao/ped#1

### LINKED ISSUE #173: [PE-D] UG bug 2
Actual output from the jar file differs from the ones in UG.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: JuZihao/ped#2

### LINKED ISSUE #174: [PE-D] UG bug 3
Actual output for project /view differs from the ones shown in UG.
e.g, [ID:] is not shown

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: JuZihao/ped#3

### PR DESCRIPTION
**Pull request by gmit22**
Fixes 
* #175 
* #174 
* #173 
* #172 
* #171 
* #168 
* #166 
* #165 
* #158 
* #157 
* #152 
* #153 
* #148 
* #147 
* #146
* #144 
* #142.

### CODE REVIEW

**Review by e0426051 — APPROVED** [review]
Looks good.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -80,14 +80,16 @@ SCRUMptious is a command-line project management software, designed for project
...
-1. Type the command `project /create -title Scrumptious -desc A java project -end 30 -sd 10` 
+1. Type the command `project /create -title Scrumptious -desc A java project -dur 30 -sd 10` 
 into the command line and press `Enter` to execute it.
-1. If the setup is correct, you can view the project by executing `project /view`:
+1. If the setup is correct, you will be able to view the project by executing `project /view`:
    ```
-   command> project /create -title Scrumptious -desc A java project -end 30 -sd 10
+   command> project /create -title Scrumptious -desc A java project -dur 30 -sd 10
    Project successfully created.
+       Title: Scrumptious
```
Comment (keke101) [inline docs/UserGuide.md:89]: This line is wrong, this is what I got when I run it
```
Project successfully created.
============================ PROJECT ============================
[ID: 1]
[Title: Scrumptious]
[Description: A java project]
[No members added]
[Project will start along with the first sprint]
[Project backlog is empty]
[There are no Sprints]
=================================================================
```

**Review by keke101 — CHANGES_REQUESTED** [review]
Everything looks good except one error still needs to be changed.

File: docs/UserGuide.md (hunk 1 above)
Comment (gmit22) [reply docs/UserGuide.md:89]: No I changed the code, so did you try running with the updated code?

Comment (keke101) [reply docs/UserGuide.md:89]: Ran it with v2.0 jar file that's why I didn't see the change. Sorry about that. Looks good to me and can be merged.

**Review by keke101 — APPROVED** [review]

**Pull request merged**